message_validator.py

```diff
@@ -18,7 +18,7 @@ def validate_message(request, username, is_inbound):
     if not 'text' in request.json:
         response =  Response('','text is missing')
         return response
-    if len(request.json['text']) < 1 or len(request.json['text']) > 120:
+    if len(request.json['text']) < 1 or len(request.json['text']) > 150:
         response =  Response('','text is invalid')
```

Comment:
**Suggestion:** The length of the text is being checked twice, once for being less than 1 and once for being more than 150. This can be simplified by using the not operator.
```suggestion
    if not 1 <= len(request.json['text']) <= 150:
        response =  Response('text is invalid', status=400)
        return response
```